internal/fuzzbinder/fuzzscraper_test.go

```diff
@@ -185,3 +164,20 @@ func TestGetGCSFileDigest(t *testing.T) {
 		t.Errorf("invalid file digest: got %v want %v", got, want)
 	}
 }
+
+func TestParseSummaryForFuzzTargetPath(t *testing.T) {
+	fuzzTarget := "apply_policy"
+	path := filepath.Join(testdataPath, coverageSummaryPath)
+	fileBytes, err := os.ReadFile(path)
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+	got, err := parseSummaryForFuzzTargetPath(fileBytes, fuzzTarget)
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+	want := "oak/fuzz/fuzz_targets/apply_policy.rs"
```

Comment:
I think we want to drop `oak` (i.e., the project name) from this path. No?

Reply:
Yes, I thin so because the definition of path is `// Path of the fuzz-target, relative to the root of the Git repository.`

Reply:
The root of the repo should not contain `oak`.

Reply:
I agree :)